### DISCUSSION TITLE
Possible bug: photoprism container do not start: panic: Error 1062: Duplicate entry

### DISCUSSION THREAD

**Discussion by Rodionc** [photoprism/photoprism#1978] [Q&A] [answered]
Hi!
I installed Photoprism and indexed my photo folder, indexing was done successfully, but after restart, docker photoprism container do not start: 
`photoprism_1  | panic: Error 1062: Duplicate entry 'Hlavní město Praha, Praha, Czech Republic' for key 'uix_places_place_label'`
I have attached to mysql db and made some investigations, which shows, that table `places` has duplicaing values in `place_label` column:
`MariaDB [photoprism]> select count(place_label) as cnt, place_label from places group by place_label order by cnt desc;`
`| cnt | place_label                                                                                                         |`
`|   5 | Washington, District of Columbia, USA                                                                               |`
`|   4 | Hlavní město Praha, Praha, Czech Republic                                                                         |`

Full execption log attached.
[photoprism_crash.txt](https://github.com/photoprism/photoprism/files/7932093/photoprism_crash.txt)

**Comment by lastzero** [comment, marked as answer]
There should be no index on this column (anymore). Did you upgrade? What version do you use?

**Reply by Rodionc** [reply]
I had an old version (near yrear old), than I call `docker-compose stop`, remove DB volume,  `docker-compose pull`, `docker-compose up -d` and then `docker-compose exec photoprism photoprism index`.

**Reply by lastzero** [reply]
When PhotoPrism starts, it typically does a database migration. If you don't wait for this and run index before, it is expected to cause issues.

**Reply by lastzero** [reply]
Look here (you can also run them manually if you feel like it doesn't happen automatically):

https://github.com/photoprism/photoprism/blob/develop/internal/migrate/dialect_mysql.go#L8
